### ISSUE TITLE
loss is in [3,4],how can I do

### ISSUE THREAD

**Issue by unshaven** [dengdan/seglink#53]
thanks for your work~
I use your 384-pre-trained model to finetune on my own dataset of 70000 images
but my loss is always around 3,can you tell me how can I do?
![image](https://user-images.githubusercontent.com/31883567/61121867-e430c680-a4d2-11e9-9f36-d98f01641a0d.png)

**Comment by zhangjunyi1225054736** [comment]
how do you solve this problem?
